conf/application.conf

```diff
@@ -98,6 +98,7 @@ auditing {
 mongodb {
   uri = "mongodb://localhost:27017/advance-valuation-rulings"
   userAnswersTtlInDays = 28
+  traderDetailsTtlInSeconds = 60
```

Comment:
should this be longer?

Reply:
For local this is good for development, I've set the value in prod to be longer https://github.com/hmrc/app-config-base/pull/8610